packages/opentelemetry-metrics/test/Meter.test.ts

```diff
@@ -205,19 +200,19 @@ describe('Meter', () => {
 
       it('should return no op metric if name is an empty string', () => {
         const counter = meter.createCounter('');
-        assert.ok(counter instanceof NoopMetric);
+        assert.ok(counter instanceof types.NoopMetric);
```

Comment:
nit: I usually expect `* as types` imports to be only used for types and to be compiled away.

Reply:
An `instanceof` cannot be compiled away because it is a runtime check. In any case, this is a test not deployed code.

Reply:
Yes sorry I should have been more clear. I am proposing renaming this to `* as core` or `* as api`, but as you've stated this is only a test so its only a nit :)

Reply:
Even if it wasn't a test, this is an `instanceof` check which is a runtime check that can't be compiled away.

Reply:
I agree. My point is that because it is **not** being compiled away, using `import * as types` here is misleading because it is being used for more than just compile-time types.